### PR TITLE
feat(acc): Alert behavior

### PR DESCRIPTION
**Pull request by jurokapsiar**
Adds role=alert or status (depending on the variant) and aria-live=polite to the root slot

### CODE REVIEW

**Review by sophieH29 — APPROVED** [review]

**Review by bmdalex — APPROVED** [review]
👍

**Review by bmdalex — APPROVED** [review]
👍  looks good to me, pls add CHANGELOG entry before merge and couple of examples if you feel it's valuable

File: packages/react/src/lib/accessibility/Behaviors/Alert/alertBehavior.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Accessibility } from '../../types'
+import alertWarningBehavior from './alertWarningBehavior'
+import alertInfoBehavior from './alertInfoBehavior'
+
+/**
+ * @description
+ * Uses 'alertWarningBehavior` for 'danger' and 'warning' variants.
+ * Uses 'alertInfoBehavior` for 'success' and 'info' variants.
+ */
+
```
Comment (layershifter) [inline packages/react/src/lib/accessibility/Behaviors/Alert/alertBehavior.ts]: Just wondered, do we really need this empty lines between function and JSDoc?

File: packages/react/test/specs/behaviors/alertBehavior-test.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { alertBehavior } from 'src/lib/accessibility'
+
+describe('AlertBehavior.ts', () => {
+  test('use alertWarningBehavior if warning prop is defined', () => {
+    const property = {
+      warning: true,
+    }
+    const expectedResult = alertBehavior(property)
```
Comment (layershifter) [inline packages/react/test/specs/behaviors/alertBehavior-test.tsx]:
```suggestion
    const expectedResult = alertBehavior({ warning: true })
```

nit: Can it be just one-line?

File: packages/react/src/lib/accessibility/Behaviors/Alert/alertInfoBehavior.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+ * @specification
+ * Adds role 'status' to 'content' component's part.
+ * Adds Adds attribute 'aria-live=polite' to 'content' component's part.
+ */
+
+const alertInfoBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    content: {
+      role: 'status',
+      ['aria-live']: 'polite',
```
Comment (layershifter) [inline packages/react/src/lib/accessibility/Behaviors/Alert/alertInfoBehavior.ts]:
```suggestion
      'aria-live': 'polite',
```

File: packages/react/src/lib/accessibility/Behaviors/Alert/alertWarningBehavior.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ * @specification
+ * Adds role 'alert' to 'content' component's part.
+ * Adds Adds attribute 'aria-live=polite' to 'content' component's part.
+ */
+
+const alertWarningBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    content: {
+      role: 'alert',
+      ['aria-live']: 'polite',
```
Comment (layershifter) [inline packages/react/src/lib/accessibility/Behaviors/Alert/alertWarningBehavior.ts]:
```suggestion
      'aria-live': 'polite',
```

**Review by layershifter — APPROVED** [review]

**Pull request merged**